.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "docfx": {
+      "version": "2.60.2",
+      "commands": [
+        "docfx"
+      ]
+    }
+  }
+}
```

Comment:
It's trivial, but could you add a new line to the end of the file?

Reply:
Amended!